lib/streamlit/cli.py

```diff
@@ -96,17 +96,31 @@ def main_hello():
 
 
 @main.command('run')
-@click.argument('file', type=click.Path(exists=True))
+@click.argument('file', type=click.Path(exists=True), required=False)
 @click.argument('args', nargs=-1)
-def main_run(file, args):
+@click.option('-m', '--module')
```

Comment:
I am adding an option to the existing run command. An alternative would be to have a completely different command. Opinions?

Reply:
I think this is good.